sync-orders-to-bigQuery/index.js

```diff
@@ -0,0 +1,66 @@
+'use strict'
+
+const { MoltinClient } = require('@moltin/request')
+const moment = require('moment')
```

Comment:
```suggestion
const moment = require('moment')

```

Reply:
What is being changed here? Sorry, I can't see the difference :)